packages/vendure-scripts/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@pinelab/vendure-scripts",
+  "version": "1.0.0",
+  "description": "Scripts that can be used with Vendure projects",
+  "icon": "script",
+  "author": "Surafel Tariku <surafelmelese09@gmail.com>",
+  "homepage": "https://pinelab-plugins.com/",
+  "repository": "https://github.com/Pinelab-studio/pinelab-vendure-plugins",
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```

Comment:
The paths are actually `dist/vendure-scripts/src/index.js` (and .d.ts) because we import utils from /test and /utils